src/server/logging/applyFilterToKey.js

```diff
@@ -0,0 +1,23 @@
+function replacer(match, group) {
+  return (new Array(group.length + 1).join('X'));
+}
+
+module.exports = function applyFilterToKey(obj, key, action) {
+  for (let k in obj)  {
+    if (obj.hasOwnProperty(k)) {
+      let val = obj[k];
+      if (k === key) {
+        val = ''  + val;
+        if (action === 'remove') delete obj[k];
+        if (action === 'censor') {
```

Comment:
I like the idea of removing if it's an object and it's set to censor. I'm also going to expose this via the config. It will make the feature more powerful.